<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="vairamuthu" ignored="1" name="vaira1" tests="2" failures="1" timestamp="18 Feb 2020 13:56:29 GMT" time="88.065" errors="0">
  <testcase name="FundTransferOtherAccountRegularCurrentAccount" time="25.532" classname="testcases.FundTransferOtherAccount">
    <failure type="java.lang.RuntimeException" message="FAILED">
      <![CDATA[java.lang.RuntimeException: FAILED
at utils.Reporter.reportStep(Reporter.java:29)
at wrappers.GenericWrappers.verifyTextByXpath(GenericWrappers.java:474)
at pages.HomePage.verifyWelcomeUserName(HomePage.java:97)
at testcases.FundTransferOtherAccount.Login(FundTransferOtherAccount.java:38)
at testcases.FundTransferOtherAccount.FundTransferOtherAccountRegularCurrentAccount(FundTransferOtherAccount.java:76)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- FundTransferOtherAccountRegularCurrentAccount -->
  <testcase name="FundTransferOtherAccountSuperValueAccount" time="50.513" classname="testcases.FundTransferOtherAccount"/>
  <testcase name="FundTransferOtherAccountAxcessAccount" time="0.0" classname="testcases.FundTransferOtherAccount">
    <ignored/>
  </testcase> <!-- FundTransferOtherAccountAxcessAccount -->
</testsuite> <!-- vaira1 -->
